<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-11T22:45:00 IST" name="LoginSuite" finished-at="2025-06-11T22:45:08 IST" duration-ms="8148">
    <groups>
    </groups>
    <test started-at="2025-06-11T22:45:00 IST" name="TestLogin InValidtest" finished-at="2025-06-11T22:45:07 IST" duration-ms="7835">
      <class name="test.LoginTest">
        <test-method is-config="true" signature="setupReport()[pri:0, instance:test.LoginTest@5a7005d]" started-at="2025-06-11T22:44:59 IST" name="setupReport" finished-at="2025-06-11T22:45:00 IST" duration-ms="472" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReport -->
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:test.LoginTest@5a7005d]" started-at="2025-06-11T22:45:00 IST" name="setUp" finished-at="2025-06-11T22:45:05 IST" duration-ms="5579" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@773c0293]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testValidLogin(java.lang.String,java.lang.String)[pri:0, instance:test.LoginTest@5a7005d]" started-at="2025-06-11T22:45:05 IST" name="testValidLogin" finished-at="2025-06-11T22:45:07 IST" duration-ms="1360" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[user1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[pass1]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Just a moment...] but found [nopCommerce demo store. Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Just a moment...] but found [nopCommerce demo store. Login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at test.LoginTest.testValidLogin(LoginTest.java:73)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidLogin -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:test.LoginTest@5a7005d]" started-at="2025-06-11T22:45:07 IST" name="tearDown" finished-at="2025-06-11T22:45:07 IST" duration-ms="761" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testValidLogin status=FAILURE method=LoginTest.testValidLogin(java.lang.String,java.lang.String)[pri:0, instance:test.LoginTest@5a7005d] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="teardownReport()[pri:0, instance:test.LoginTest@5a7005d]" started-at="2025-06-11T22:45:08 IST" name="teardownReport" finished-at="2025-06-11T22:45:09 IST" duration-ms="919" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownReport -->
      </class> <!-- test.LoginTest -->
    </test> <!-- TestLogin InValidtest -->
    <test started-at="2025-06-11T22:45:00 IST" name="TestLogin Validtest" finished-at="2025-06-11T22:45:08 IST" duration-ms="8148">
      <class name="test.LoginTest">
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:test.LoginTest@13f17eb4]" started-at="2025-06-11T22:45:00 IST" name="setUp" finished-at="2025-06-11T22:45:06 IST" duration-ms="5811" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@3766c667]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testValidLogin(java.lang.String,java.lang.String)[pri:0, instance:test.LoginTest@13f17eb4]" started-at="2025-06-11T22:45:06 IST" name="testValidLogin" finished-at="2025-06-11T22:45:07 IST" duration-ms="1843" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin@yourstore.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidLogin -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:test.LoginTest@13f17eb4]" started-at="2025-06-11T22:45:07 IST" name="tearDown" finished-at="2025-06-11T22:45:08 IST" duration-ms="369" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testValidLogin status=SUCCESS method=LoginTest.testValidLogin(java.lang.String,java.lang.String)[pri:0, instance:test.LoginTest@13f17eb4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.LoginTest -->
    </test> <!-- TestLogin Validtest -->
  </suite> <!-- LoginSuite -->
</testng-results>
